doc/api/fs.md

```diff
@@ -436,7 +436,7 @@ changes:
 -->
 
 * `path` {string|Buffer|URL}
-* `mode` {integer}
+* `mode` {integer} **Default:** `fs.constants.F_OK`
```

Comment:
Yea, in general I'm good with improved consistency and won't block this, I just don't prefer the bold styling.